CalibCalorimetry/CastorCalib/src/CastorDbHardcode.cc

```diff
@@ -378,7 +378,7 @@ void CastorDbHardcode::makeHardcodeMap(CastorElectronicsMap& emap) {
 	      if (ieta==0) { // unmapped 
 		emap.mapEId2chId(elId,DetId(HcalDetId::Undefined));
 	      } else {
-		HcalDetId hId(HcalOuter,ieta*iside,iphi,idepth+3); // HO is officially "depth=4"
+		HcalDetId hId(HcalOuter,ieta*iside,iphi,4); // HO is officially "depth=4"
```

Comment:
@bsunanda the meaning of the magic number `4`, representing HO depth, is explained in the comment to the code itself. Anyhow, IIUC, it can be expressed using the masks available in `HcalDetId` and the `HcalSubdetector` enum. But you may comment further...